spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
Well, as you can see from the macro-expansion above, it is unsafe to call `volatile-highlights-mode` in `init`, regardless of what the readme says. If the package fails to install, the init form can still run and emacs' startup will be interrupted with a void-function error (as I experienced this morning). This is never a problem with `config`.